hack/update-import-aliases.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+GOIMPORTS_PKG="golang.org/x/tools/cmd/goimports"
+GOIMPORTS_VERSION="v0.1.5"
```

Comment:
Are we using `goimports` yet? I can't find a place to call it

Reply:
golangci-lint support [`goimports`](https://golangci-lint.run/usage/linters/#goimports)，but goimports is disabled by default, I forgot to enable it

Reply:
You can enable goimports in a new pr, It does not seem to be related to this pr.

Reply:
yes , I want to add some linters in .golangci.yml for code check in next pr . Do you think it is ok ?
golangci-lint seems only check the goimports but not rewrite

Reply:
> You can enable goimports in a new pr, It does not seem to be related to this pr.

yes  I removed it